pkg/render/common/networkpolicy/networkpolicy.go

```diff
@@ -290,6 +290,10 @@ func (h *NetworkPolicyHelper) ComplianceReporterSourceEntityRule() v3.EntityRule
 	return CreateSourceEntityRule(h.namespace("tigera-compliance"), "compliance-reporter")
 }
 
+func (h *NetworkPolicyHelper) KibanaEntityRule() v3.EntityRule {
```

Comment:
I think this function is only used for Multi-tenant kibana and in that case doesn't this rule need to use the tenant namespace?